scss/tech.scss

```diff
@@ -0,0 +1,44 @@
+.tech-content {
+  margin: 0 auto;
+  padding: 32px 170px;
+}
+
+.tech {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  justify-content: flex-start;
+  padding: 16px 40px 40px 16px;
```

Comment:
`padding: top right bottom left` this padding is weird also, what are you trying to do?